src/main/resources/application.yml

```diff
@@ -40,8 +40,8 @@ spring:
     max-in-memory-size: 24MB
   data:
     mongodb:
-      uri: ${MONGO_URI:mongodb://${DBUSER:root}:${DBPASS:E5press0}@localhost:27017/terminology}?authSource=admin&maxPoolSize=50&connectTimeoutMS=2000&serverSelectionTimeoutMS=2000
+      uri: mongodb://root:E5press0@${MONGO_HOST:localhost}:27017/madie?authSource=admin&maxPoolSize=50&connectTimeoutMS=2000&serverSelectionTimeoutMS=2000
```

Comment:
was this intentionally added? Should we maybe make `root` a default value for the `${DBUSER}` var? (And same for `DBPASS`)

Reply:
Nope! Accidental add. I'll clear it out.